README.md

```diff
@@ -119,6 +119,7 @@ These environment variables are required on the following suites:
 - bumpzone
 - cookbook-uploader
 - github-comment
+- ibmz-ci
```

Comment:
This suite also needs OAuth authentication, and should be mentioned in the section below